dbt_project.yml

```diff
@@ -54,7 +54,7 @@ models:
 
 vars:
   dbt_date:time_zone: Etc/UTC
-  fullstory_incremental_interval: INTERVAL 8 DAY
+  fullstory_incremental_interval_hours: 192 # 8 days
```

Comment:
Is this a breaking change? Does it need to up-version?
(Or not since it never worked?)

Also seems to need to update: https://github.com/fullstorydev/dbt_fullstory/blob/9b2cbe5be5ee53630ab93821bfc62a872eb249a5/README.md?plain=1#L214

Reply:
Yes, planning to increment the version after this is merged. Will update the readme now!